### PR TITLE
cmd/action: adding --addr flag, small copy changes

### PR DESCRIPTION
**Pull request by rotemtam**

### CODE REVIEW

**Review by yonidavidson — APPROVED** [review]

File: cmd/action/apply.go [hunk 1]
```diff
@@ -14,9 +14,10 @@ import (
 var (
 	// ApplyFlags are the flags used in Apply command.
 	ApplyFlags struct {
-		DSN  string
-		File string
-		Web  bool
+		DSN     string
+		File    string
+		Web     bool
+		Address string
```
Comment (masseelch) [inline cmd/action/apply.go:20]: I'd call it `Port` tbh

File: cmd/action/apply.go [hunk 2]
```diff
@@ -40,15 +41,16 @@ func init() {
 	schemaCmd.AddCommand(ApplyCmd)
 	ApplyCmd.Flags().StringVarP(&ApplyFlags.DSN, "dsn", "d", "", "[driver://username:password@protocol(address)/dbname?param=value] Select data source using the dsn format")
 	ApplyCmd.Flags().StringVarP(&ApplyFlags.File, "file", "f", "", "[/path/to/file] file containing schema")
-	ApplyCmd.Flags().BoolVarP(&ApplyFlags.Web, "web", "w", false, "open in UI server")
+	ApplyCmd.Flags().BoolVarP(&ApplyFlags.Web, "web", "w", false, "Open in a local Atlas UI")
+	ApplyCmd.Flags().StringVarP(&ApplyFlags.Address, "address", "", "127.0.0.1:5800", "used with -w, local address to bind the server to")
```
Comment (masseelch) [inline cmd/action/apply.go:45]: dito

File: cmd/action/apply.go [hunk 3]
```diff
@@ -40,15 +41,16 @@ func init() {
...
 	cobra.CheckErr(ApplyCmd.MarkFlagRequired("dsn"))
 	cobra.CheckErr(ApplyCmd.MarkFlagRequired("file"))
 }
 
 // CmdApplyRun is the command used when running CLI.
 func CmdApplyRun(cmd *cobra.Command, args []string) {
 	if ApplyFlags.Web {
-		schemaCmd.PrintErrln("Opening the Graphical UI server is not available in this release")
+		schemaCmd.PrintErrln("The Atlas UI is not available in this release.")
```
Comment (masseelch) [inline cmd/action/apply.go:53]: :+1:

File: cmd/action/inspect.go [hunk 4]
```diff
@@ -9,8 +9,9 @@ import (
 var (
 	// InspectFlags are the flags used in Inspect command.
 	InspectFlags struct {
-		DSN string
-		Web bool
+		DSN     string
+		Web     bool
+		Address string
```
Comment (masseelch) [inline cmd/action/inspect.go:14]: Same.

File: doc/md/CLI/atlas_schema_apply.md [hunk 5]
```diff
@@ -19,10 +19,11 @@ atlas schema apply -d "sqlite://file:ex1.db?_fk=1" -f atlas.hcl
 ### Options
 
 ```
-  -d, --dsn string    [driver://username:password@protocol(address)/dbname?param=value] Select data source using the dsn format
-  -f, --file string   [/path/to/file] file containing schema
-  -h, --help          help for apply
-  -w, --web           open in UI server
+      --address string   used with -w, local address to bind the server to (default "127.0.0.1:5800")
```
Comment (masseelch) [inline doc/md/CLI/atlas_schema_apply.md:22]: Same here.

File: doc/md/CLI/atlas_schema_inspect.md [hunk 6]
```diff
@@ -19,9 +19,10 @@ atlas schema inspect -d "sqlite://file:ex1.db?_fk=1"
 ### Options
 
 ```
-  -d, --dsn string   [driver://username:password@protocol(address)/dbname?param=value] Select data source using the dsn format
-  -h, --help         help for inspect
-  -w, --web          open in UI server
+      --address string   used with -w, local address to bind the server to (default "127.0.0.1:5800")
```
Comment (masseelch) [inline doc/md/CLI/atlas_schema_inspect.md:22]: and again ;-)

**Review by masseelch — APPROVED** [review]

File: cmd/action/apply.go (hunk 1 above)
Comment (a8m) [reply cmd/action/apply.go:20]: I'd call it `--addr` as it's passed as-is to `Listen`.

**Pull request merged**